packages/base/src/Base.ts

```diff
@@ -163,3 +163,49 @@ export function SpectrumMixin<T extends Constructor<ReactiveElement>>(
 }
 
 export class SpectrumElement extends SpectrumMixin(LitElement) {}
+
+if (window.__swc.DEBUG) {
+    window.__swc = {
```

Comment:
Curious why attach them to the window object and not make these available to call on the component.